dashboard/utils.py

```diff
@@ -277,6 +282,47 @@ def count_courses_passed(self):
                 passed_courses.add(self.edx_key_course_map[course_id])
         return len(passed_courses)
 
+    def get_pearson_exam_status(self):  # pylint: disable=too-many-return-statements
+        """
+        Get the pearson exam status for the user / program combo
+        """
+        if self.program.exam_series_code is None:
```

Comment:
Line 30 in the linked code, would it be hepful if I addressed this in a small upstream PR?